src/core/core.helpers.js

```diff
@@ -431,6 +431,19 @@ module.exports = function(Chart) {
 		var width = boundingRect.right - boundingRect.left - paddingLeft - paddingRight;
 		var height = boundingRect.bottom - boundingRect.top - paddingTop - paddingBottom;
 
+		if (/chrome/i.test(navigator.userAgent)) {
+			// In Chrome zoom of a parent node requires to adjust mouseX/mouseY
+			var parent = canvas.parentNode;
+			while (parent.parentNode !== null) {
```

Comment:
would it make sense to have this be `while (parent !== null)`?

Reply:
I checked for ```parent.parentNode !== null``` because the highest parent is a ```document``` node, so I wanted to stop before.